site/docs/v1/tech/samlv2/index.adoc

```diff
@@ -198,4 +198,10 @@ Of course, you can modify, delete, or add any attributes you want by configuring
 
 == Limitations
 
+=== IdP Limitations
```

Comment:
The Limitations page had these two sections, but only one was displayed on the SAML page. I assumed it was an oversight, but let m know if I should roll this back.

Reply:
Thanks, appreciate it.